### PR TITLE
Add option of editing the .bashrc

### PR DESCRIPTION
**Pull request by denCalderon**
Add an option  asking the user if they would like to edit their configuration file. If not then we can tell the user what lines they need to add.

### CODE REVIEW

File: kit/commands/init.py [hunk 1]
```diff
@@ -126,34 +126,53 @@ def create_plugin_data(dir_path: Path) -> None:
         print(f"{plugin_file_path} created")
 
 
+def get_rc_new_lines() -> str:
+    """Return the lines that will be added in the rc file"""
```
Comment (jlhcrawford) [inline kit/commands/init.py:130]:
```suggestion
    """Return the lines that will be added to the rc file"""
```

File: kit/commands/init.py [hunk 2]
```diff
@@ -126,34 +126,53 @@ def create_plugin_data(dir_path: Path) -> None:
...
     rc_path = get_expanded_path(rc_file)
+
+    user_answer = input(
+        f"The hekit init command will update the file {rc_file} to append the following lines:\n\n"
+        f"{rc_new_content}\n"
+        "Do you want to continue with this action? (y/n) "
+    )
+    if user_answer not in ("y", "Y"):
+        print(
+            f"Please, execute the following actions manually:\n"
```
Comment (jlhcrawford) [inline kit/commands/init.py:161]:
```suggestion
            f"Please execute the following actions manually:\n"
```

File: kit/commands/init.py [hunk 3]
```diff
@@ -126,34 +126,53 @@ def create_plugin_data(dir_path: Path) -> None:
...
+            f"1. Open the file {rc_file}\n"
+            "2. Add the lines shown in the previous message\n"
+            f"3. Source your shell init file as: source {rc_file}"
```
Comment (jlhcrawford) [inline kit/commands/init.py:164]:
```suggestion
            f"3. Source your shell config file with: source {rc_file}"
```

File: kit/commands/init.py [hunk 4]
```diff
@@ -126,34 +126,53 @@ def create_plugin_data(dir_path: Path) -> None:
...
+        )
+        return
+
+    # Backup the shell init file
```
Comment (jlhcrawford) [inline kit/commands/init.py:168]:
```suggestion
    # Backup the shell config file
```

File: tests/test_integration_init.py [hunk 5]
```diff
@@ -23,8 +23,22 @@ def test_init_hekit_create_config_file(mocker, tmp_path):
...
+    mockers = Mockers(mocker, tmp_path, default_config_flag=True)
+    mockers.mock_input.return_value = "n"
+    mockers.create_tmp_config_file()
+    # Create a new config file
+    main()
+    mockers.mock_print.assert_any_call(
+        "Please, execute the following actions manually:\n"
+        f"1. Open the file {tmp_path}/.mybashfile\n"
+        "2. Add the lines shown in the previous message"
+        f"\n3. Source your shell init file as: source {tmp_path}/.mybashfile"
```
Comment (jlhcrawford) [inline tests/test_integration_init.py:36]:
```suggestion
        f"\n3. Source your shell config file with: source {tmp_path}/.mybashfile"
```

File: kit/commands/init.py [hunk 6]
```diff
@@ -126,34 +126,53 @@ def create_plugin_data(dir_path: Path) -> None:
...
         create_plugin_data(dir_path)
 
-    # Backup the shell init file
+    rc_new_content = get_rc_new_lines()
     rc_file = get_rc_file()
     rc_path = get_expanded_path(rc_file)
+
+    user_answer = input(
+        f"The hekit init command will update the file {rc_file} to append the following lines:\n\n"
+        f"{rc_new_content}\n"
```
Comment (jlhcrawford) [inline kit/commands/init.py:156]:
```suggestion
        f"{rc_new_content}\n"
        "NOTE: a backup file will be created.\n"
```

**Review by jlhcrawford — CHANGES_REQUESTED** [review]
This looks very good. Just some very minor tweaks.

File: kit/commands/init.py (hunk 2 above)
Comment (denCalderon) [reply kit/commands/init.py:161]: updated

File: kit/commands/init.py (hunk 3 above)
Comment (denCalderon) [reply kit/commands/init.py:164]: updated

File: kit/commands/init.py (hunk 4 above)
Comment (denCalderon) [reply kit/commands/init.py:168]: updated

File: kit/commands/init.py (hunk 6 above)
Comment (denCalderon) [reply kit/commands/init.py:156]: updated

File: kit/commands/init.py [hunk 7]
```diff
@@ -73,7 +73,7 @@ def append_to_rc(path: Path, content: str) -> None:
 
     # newline to not accidentally mix with existing content
     # newline to end as courtesy to space our code
-    lines = ["\n", Tags.start_tag, content, Tags.end_tag, "\n"]
+    lines = ["\n", content, "\n"]
```
Comment (hamishun) [inline kit/commands/init.py:76]: Why are the tags removed?

File: kit/commands/init.py [hunk 8]
```diff
@@ -126,37 +126,57 @@ def create_plugin_data(dir_path: Path) -> None:
...
+    """Return the lines that will be added to the rc file"""
+    # 1-Add hekit directory as part of environmental variable PATH
+    export_line = f"export HEKITPATH={Constants.HEKIT_ROOT_DIR}\n"
+    path_line = "PATH=$PATH:$HEKITPATH\n"
+    # 2-Register hekit link and hekit.py script to enable tab completion
+    eval_lines = (
+        "if [ -n $(type -p register-python-argcomplete) ]; then\n"
+        '  eval "$(register-python-argcomplete hekit)"\n'
+        "fi\n"
+    )
```
Comment (hamishun) [inline kit/commands/init.py:139]:
```suggestion
    eval_lines = (
        'if [ -n "$(type -p register-python-argcomplete)" ]; then\n'
        '  eval "$(register-python-argcomplete hekit)"\n'
        'fi\n'
    )
```

File: kit/commands/init.py [hunk 9]
```diff
@@ -126,37 +126,57 @@ def create_plugin_data(dir_path: Path) -> None:
...
+
+    user_answer = input(
+        f"The hekit init command will update the file {rc_file} to append the following lines:\n\n"
+        f"{rc_new_content}\n"
+        "NOTE: a backup file will be created before updating it.\n"
+        "Do you want to continue with this action? (y/n) "
+    )
+    if user_answer not in ("y", "Y"):
+        print(
+            f"Please execute the following actions manually:\n"
```
Comment (hamishun) [inline kit/commands/init.py:162]:
```suggestion
            "Please execute the following actions manually:\n"
```

**Review by hamishun — CHANGES_REQUESTED** [review]

File: kit/commands/init.py (hunk 7 above)
Comment (hamishun) [reply kit/commands/init.py:76]: I see the are inserted elsewhere, ok

File: kit/commands/init.py [hunk 10]
```diff
@@ -126,37 +126,57 @@ def create_plugin_data(dir_path: Path) -> None:
...
-        '  eval "$(register-python-argcomplete hekit)"\n'
-        "fi\n"
-    )
-    content = "".join([export_line, path_line, eval_lines])
-    append_to_rc(rc_path, content)
+    append_to_rc(rc_path, rc_new_content)
 
     # Instructions for user
-    print("Please, source your shell init file as follows")
+    print("Please, source your shell config file as follows")
```
Comment (jlhcrawford) [inline kit/commands/init.py:179]:
```suggestion
    print("Please source your shell config file as follows")
```

**Review by jlhcrawford — APPROVED** [review]

**Review by hamishun — APPROVED** [review]

**Pull request merged**